.ci/thin_macos_qtlib.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# The macos binaries from aqt are fat (universal), so we thin them to the target architecture to reduce the size of
+# the packages and caches using lipo.
+
+# This script is meant to be used by the ci enironment on macos runners only.
+# It uses the runner's GITHUB_WORKSPACE env variable.
+arch=$(uname -m)
+nproc=$(sysctl -n hw.ncpu)
+
+function thin() {
+  local libfile=$1
+  if [[ $(file -b --mime-type "$libfile") == application/x-mach-binary ]]; then
```

Comment:
```suggestion
  if [[ $(file -b --mime-type "$libfile") == application/x-mach-binary* ]]; then
```

it just starts with this text